redis/reply_test.go

```diff
@@ -135,6 +151,61 @@ func TestReply(t *testing.T) {
 	}
 }
 
+func TestSlowLog(t *testing.T) {
+	c, err := dial()
+	if err != nil {
+		t.Errorf("TestSlowLog failed during dial with error " + err.Error())
+		return
+	}
+	defer c.Close()
+
+	result, err := c.Do("CONFIG", "GET", "SLOWLOG-LOG-SLOWER-THAN")
+	if err != nil {
+		t.Errorf("TestSlowLog failed during CONFIG GET SLOWLOG-LOG-SLOWER-THAN with error " + err.Error())
+		return
+	}
+	resultStr, err := redis.Strings(result, err)
```

Comment:
no need to separate the strings call, you can merge into the line above.

Reply:
Fixed it.